### PR TITLE
Sowmika, Vinay | MOBN-636 | Fix partial write of obs data for forms2

### PR DESCRIPTION
**Pull request by sowmika148**

### CODE REVIEW

File: src/main/java/org/bahmni/mart/exports/Form2ObservationExportStep.java [hunk 1]
```diff
@@ -136,7 +131,7 @@ private String getFormName() {
     public Step getRemovalStep() {
         return stepBuilderFactory.get(getStepName("Removal Step"))
                 .<Map<String, Object>, Map<String, Object>>chunk(100)
-                .reader(obsReader(true))
+                .reader(obsReader())
```
Comment (shivarach) [inline src/main/java/org/bahmni/mart/exports/Form2ObservationExportStep.java:134]: Removal step supposed to remove voided records only. Is it doing same?

**Review by shivarach — CHANGES_REQUESTED** [review]
Check the comments

File: src/main/java/org/bahmni/mart/exports/Form2ObservationExportStep.java (hunk 1 above)
Comment (sowmika148) [reply src/main/java/org/bahmni/mart/exports/Form2ObservationExportStep.java:134]: Discussed with Siva

**Comment by shivarach** [comment]
Looks good to me.

**Pull request merged**